docs/source/notes/faq.rst

```diff
@@ -18,57 +26,70 @@ please remove the installation directory from your MATLAB path.
     >> savepath % save the new path
     >> delete(CBTDIR,'s') % delete the installation directory
 
-Github
-------
 
-**What do all these labels mean?**
+When running ``git submodule update``, the following error message appears. What should I do?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A comprehensive list of labels and their description for the issues and
-pull requests is given
-`here <https://opencobra.github.io/cobratoolbox/docs/labels.html>`__.
+.. code::
 
-Reconstruction
---------------
+    No submodule mapping found in .gitmodules for path 'external/lusolMex64bit'
+
+remove the cached version of the respective submodule by typing:
+
+.. code::
```

Comment:
`MATLAB` code has to be within a `.. code-block:: matlab` instead of `.. code::`.